libraries/codegen-tooling/src/tmp.c

```diff
@@ -0,0 +1 @@
+// Dummy file to ensure the build succeeds.
```

Comment:
Why do we need this file?

Reply:
For some reason the build system won't build the library if there are no `*.c` files in it.....

Reply:
I think we're explicitly searching for `.c` or `.h` files in `make/build.mk` to build. You can probably add a new `make` target and then add that as a dependency for the other targets?

It's also probably fine if you want to merge as is so we can unblock work that relies on this.

Reply:
I'll merge as is and we can fix this at a later date.

Reply:
Sure. Can you create a ticket and add it to our backlog?